chalice/app.py

```diff
@@ -7,12 +7,12 @@
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'chalicelib'))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
-from dss import BucketStage, Config, create_app
+from dss import DeploymentStage, Config, create_app
```

Comment:
I, too, like the rename, but this clutters the diff up significantly.  This should be done in a separate PR.  #290